### PR TITLE
Configurable idle periods

### PR DESCRIPTION
**Pull request by shortbloke**
Add ability to make idle time configurable, as 25% is too long for longer running stress tests. 
This does change the meaning of duration in the command line, to be the actual stress test duration, not 50% of it.
The defaults are set so that they have the same time periods as before this PR.
If using --duration however, then this has changed.

### CODE REVIEW

**Comment by nschloe** [comment]
Alright, some conflicts. Care to resolve?

**Comment by shortbloke** [comment]
Sure thing. Working on it now.

File: stressberry/cli.py [hunk 1]
```diff
@@ -50,8 +50,22 @@ def _get_parser_run():
         "-d",
         "--duration",
         type=int,
-        default=600,
-        help="test duration in seconds (default: 600)",
+        default=300,
```
Comment (nschloe) [inline stressberry/cli.py]: Don't change the default.

**Comment by shortbloke** [comment]
@nschloe Should all be good now.

**Pull request merged**